src/components/user/user.service.ts

```diff
@@ -860,28 +523,10 @@ export class UserService {
     userId: ID,
     session: Session
   ): Promise<KnownLanguage[]> {
-    const results = await this.db
-      .query()
-      .match([
-        requestingUser(session),
-        ...permissionsOfNode('Language'),
-        relation('in', 'knownLanguageRel', 'knownLanguage', { active: true }),
-        node('user', 'User', { id: userId }),
-      ])
-      .with('collect(distinct user) as users, node, knownLanguageRel')
-      .raw(`unwind users as user`)
-      .return([
-        'knownLanguageRel.value as languageProficiency',
-        'node.id as languageId',
-      ])
-      .asResult<{
-        languageProficiency: LanguageProficiency;
-        languageId: ID;
-      }>()
-      .run();
+    const results = await this.userRepo.listKnownLanguages(userId, session);
 
     const knownLanguages = await Promise.all(
-      results.map(async (item) => {
+      results.map(async (item: any) => {
```

Comment:
Should probably clean up these `any`s at some point too. Probably need to define return types for the repo methods.